lib/src/main/java/ru/nsu/fit/XMLSerialization/XMLSerializator.java

```diff
@@ -0,0 +1,129 @@
+package ru.nsu.fit.XMLSerialization;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.Type;
+import java.util.*;
+import java.util.stream.Stream;
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.transform.*;
+import javax.xml.transform.dom.DOMSource;
+import javax.xml.transform.stream.StreamResult;
+
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+
+public class XMLSerializator {
+  private int id = 0;
+  private static final List<String> primAndWrappers =
```

Comment:
Change to Set (`contains` for hashset is O(1) vs O(n) for list)

Reply:
fixed